tests/01-basic.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+CURR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+[ -d "$CURR_DIR" ] || { echo "FATAL: no current dir (maybe running in zsh?)";  exit 1; }
+
+# shellcheck source=./common.sh
+source "$CURR_DIR/common.sh"
+
+#########################################################################################
+
+function finish {
+  info "Cleaning up any leftovers..."
+  $EXE delete --all > /dev/null 2>&1 || /bin/true
```

Comment:
This can be dangerous if someone is running this on his/her local machine (not inside dind).

Reply:
I see... Then I will remove it. At some moment it would be good to have some `--filter` argument...